polls/views.py

```diff
@@ -11,18 +12,35 @@ class IndexView(generic.ListView):
     context_object_name = 'latest_question_list'
 
     def get_queryset(self):
-        return Question.objects.order_by('-pub_date')[:5]
+        return Question.objects.filter(pub_date__lte=timezone.now()).exclude(
+            choice__isnull=True).order_by('-pub_date').distinct()[:5]
```

Comment:
I was looking for a way to exclude questions without choices, tried these two solutions and forgot to unify them. Is there a preferred way to do this?